charts/opentelemetry-collector/values.yaml

```diff
@@ -449,6 +449,20 @@ collector:
       log:
         enable: true
         level: debug
+      kafka:
+        enable: false
+        brokers:
+          - kafka:9093
+        encoding: otlp_proto
+        auth:
```

Comment:
We should have an option to bring it from some secret and not only plain text, don't you think ?

Reply:
I support brining secrets from vault by providing the secret vault path (e.g. `vault:secret/path_to_secret/#KAFKA_USERNAME`).
But sure, I can support other secrets as well.